libs/features/mas/src/merch-card-collection.js

```diff
@@ -156,20 +156,19 @@ export class MerchCardCollection extends LitElement {
             this.hasMore = result.length > pageSize;
             result = result.filter(([, index]) => index < pageSize);
         }
-
-        let reduced = new Map(result);
+        let reduced = new Map(result.reverse());
+        for (const card of reduced.keys()) {
+          this.prepend(card);
+        }
```

Comment:
doesn't this work and would be simpler?
```suggestion
        let reduced = new Map(result());
        for (const card of reduced.keys()) {
          this.appendChild(card);
        }
```

Reply:
doesn't look like it would work at all :-p

Reply:
yeah visible cards need to be in the beginning